AK/bits/Math/common.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022, Leon Albrecht <leon2002.la@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
```

Comment:
I'm not sure i'm getting why the path AK/bits/Math is necessary? Seems a flatter structure would do the trick? The bits/ thing seems inherited from glibc. Math would be the only subdirectory of the new `bits` directory. Other files have done like, FooFeature.h and FooFeatureDetails.h.

Reply:
The existence of this file is also confusing. It seems like the only times it's actually used in the arch-specific headers is when someone like HeaderCheck includes the "arch impl" headers directly? Otherwise it's already there from Math.h.

Reply:
I put this here, because otherwise clangd will complain about non defined constants etc....